build.gradle

```diff
@@ -667,6 +675,17 @@ task genFakeSource(dependsOn: ["buildYarnTiny", "mapNamedJar"]) {
 	}
 }
 
+task decompileCFR(type: JavaExec, dependsOn: "mapNamedJar") {
+	classpath = configurations.decompileClasspath
+	main = "org.benf.cfr.reader.Main"
+
+	args namedJar.getAbsolutePath(), "--outputdir", file("src").absolutePath
+
+	doFirst {
+		file("src").deleteDir()
```

Comment:
Shouldn't this be `minecraftSrc` or similar to be in line with `remappedSrc`, and in certain scenarios, not overwrite an existing `src` folder?

Reply:
there isnt an existing src, I did kinda think of dumping it in src/main/java and making it importable into an ide so you could explore the code a bit easier but thats prob not needed.

I think what we have here is fine? What do others think?

Reply:
on 2nd throughts, remappedSrc does make more sense. I will change this

Reply:
Another solution would be to potentially halt the process if `src` already exists.

Reply:
changed to `namedSrc` to follow the jar.